### ISSUE TITLE
bug(YouTube - Hide ads): "Sponsored" advertisements are shown

### ISSUE THREAD

**Issue by TheSubMatrix** [ReVanced/revanced-patches#7]
### Bug description

On the latest recommended patch of YouTube (18.45.43) patched on the latest release version of revanced manager (1.17.3) with root patching, I am beginning to see sponsored advertisements popping up on the home page and in my recommended. The advertisements do not appear to being correctly blocked by the existing patches. 
![Screenshot_2023-12-12-23-50-54-28_f9ee0578fe1cc94de7482bd41accb329](https://github.com/ReVanced/revanced-patches/assets/25988883/857b84c2-38f0-4af0-a621-7cd4c128e25e)

The issues began sometime earlier today on an older patched version of vanced. I updated to see if perhaps the error was resolved on a future patch and this did not appear to fix the issue.

Below are the logcat files from the bug.
[13_12-00-47-58_889.log](https://github.com/ReVanced/revanced-patches/files/13656760/13_12-00-47-58_889.log)

Below is my patch list
[selected_patches_2023-12-13_00_15_11.json](https://github.com/ReVanced/revanced-patches/files/13656764/selected_patches_2023-12-13_00_15_11.json)

### Error logs

```shell
Listed in the above description
```

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by johnconner122** [comment]
Also try `full_width_portrait_image_layout`.

**Comment by oSumAtrIX** [comment]
Follow this and report back with the full line of string:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/7ce0abbd-af82-4200-8595-f7fc8ebb173e)

**Comment by LtPatterson** [comment]
Applied both filters, appears to be working. Need additional confirmation from someone else to be sure.

**Comment by TheSubMatrix** [comment]
Filters appear to be working for me!

**Comment by johnconner122** [comment]
> Filters appear to be working for me!

Which filter solved your issue?

**Comment by LtPatterson** [comment]
> > Filters appear to be working for me!
> 
> Which filter solved your issue?

Either filter works, as well as combining both filters. If you disable the filters, the sponsored posts appear again.

**Comment by sinkaf** [comment]
> > Filters appear to be working for me!
> 
> Which filter solved your issue?

When the filter you first shared was applied, the ads disappeared. When I applied the second filter you shared alone, ads appeared and I didn't understand what it was for, so I deleted it.
